src/transactions/OfferExchange.cpp

```diff
@@ -967,6 +1011,38 @@ OfferExchange::crossOfferV10(OfferFrame& sellingWheatOffer,
         sellingWheatOffer.storeChange(mDelta, db);
     }
 
+    if (mLedgerManager.getCurrentLedgerVersion() >= 11)
+    {
+        int64_t deltaBuying = sellingWheatOffer.getBuyingLiabilities();
+        if (sellingWheatOffer.getBuying().type() == ASSET_TYPE_NATIVE)
+        {
+            bool res = accountB->addBuyingLiabilities(deltaBuying);
+            assert(res);
```

Comment:
same thing with those `assert` calls - I know it should not happen, but better safe than sorry and throw instead.